portfolio/src/main/webapp/script.js

```diff
@@ -15,9 +15,10 @@
 
 function getComments() {
     fetch('/data').then(response => response.json()).then((comment) => {
-        const commentList = document.getElementById('greeting-container');
-        commentList.innerHTML = '';
-        commentList.appendChild(createListElement(comment));
+        const commentList = document.getElementById('comnt-list');
```

Comment:
Will the comments show up in different lines?

Reply:
Each comment will have it's own separate box which contains their name and comment.